kube_metrics_server/assets/dashboards/overview.json

```diff
@@ -0,0 +1,582 @@
+{
```

Comment:
This needs to be added to the manifest.json file of this integration. Can we do this in a separate PR though, it'd be cleaner 😄

Reply:
yep! ill remove